app/frontend/src/components/Divider.tsx

```diff
@@ -1,16 +1,38 @@
 import { Divider as MuiDivider } from "@material-ui/core";
+import classNames from "classnames";
 import React from "react";
 import makeStyles from "utils/makeStyles";
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    marginBottom: theme.spacing(1),
-    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(2),
+  },
+
+  marginTop3: {
+    marginTop: theme.spacing(3),
+  },
+
+  marginBottom3: {
+    marginBottom: theme.spacing(3),
   },
 }));
 
-export default function Divider() {
+export interface DividerProps {
+  mb3?: boolean;
+  mt3?: boolean;
```

Comment:
After thinking about it a bit more, I think just use `extraMarginTop` or bottom like your class name. We want to avoid using utility classes so I don't think this naming makes sense - especially considering there is no `mb2` or `mb1`, and it isn't clear for people who haven't used TailwindCSS before.

Reply:
In that case we should use `className`.

I know there are lots of benefits to utility classes, but we have decided earlier on that in this app it makes more sense (or maybe, it was the overall preference) to stick with JSS classes.

If you feel strongly about it, we can see what people think on slack. It's certainly not very clear-cut in this case.

Reply:
So are we just talking about naming, isn't it? Would it be OK marginBottom1, marginBottom2, marginBottom3, etc?

Reply:
Just `extraMarginBottom` and if more options are needed, use `className`

Reply:
Sorry if I'm missing your point here :S, but what do you mean as "use `className`"? Could you give an example?

Reply:
I haven't been very clear, sorry. I think there's two things:

Components should be self-contained, and part of this is not usually setting their own margin. This is because when building layouts, if there's any variation needed in margins, this will have to happen outside the component anyway.

The other part is that when we are building page layouts, we use jss styling, which is the `const useStyles = makeStyles({ exampleClass: { marginTop: ...` and then in the page layout, `const classes = useStyles(); return ...<MyComponent className={classes.exampleClass}>...`. There are things equivalent to utility classes in Material-UI we could use (the `Box` component), but we previously decided against using them because it was the overall preference not to have styling in two places (even if it's the same file).

Taking those two things into account, I'm okay with having `<Divider extraMarginBottom>` if we don't usually need to customize the extra margin amount (seems you all used the same amount in this PR), but if we start needing `<Divider extraMarginBottom={2}>` etc, to me that's just building layouts and should use jss classes like `<Divider className={classes.myDivider}>` (or more likely, a class on the components *around* the divider).

Reply:
I'd rather put the class outside as you say, but it gives me an error (don't know if I'm doing something wrong here)
![Captura de pantalla de 2021-05-27 08-42-04](https://user-images.githubusercontent.com/8875832/119778319-a6bb6380-bec7-11eb-9d70-9fad84525d65.png)

Reply:
You'll need to change the custom `Divider` component so it accepts a `className` prop and passes it through

Reply:
Is there any working example of this in another component? I tried to do that first thing and I wasn't able to accomplish that.

Reply:
https://github.com/Couchers-org/couchers/blob/48594f38354b19cca66ae5f983b80ed9b77ebd6b/app/frontend/src/components/Alert.tsx#L35 

Note in this component, the type of the props extends `MuiAlertProps`, you can just add `className` directly to the interface instead.